Data/IntMap.hs

```diff
@@ -10,6 +10,7 @@
 -- Module      :  Data.IntMap
 -- Copyright   :  (c) Daan Leijen 2002
 --                (c) Andriy Palamarchuk 2008
+--                (c) Jonathan S. 2016
```

Comment:
Yeah, I figured so, and in general I wanted to be conservative both to properly credit the work done on documentation and to avoid legal issues since I know basically nothing real about copyright law. I'll add some clarification, but leave both existing lines.